### ISSUE TITLE
If cbt detects nailgun isn't there, it probably shouldn't print nailgun related messages

### ISSUE THREAD

**Issue by farmdawgnation** [cvogt/cbt#34]
Ran cbt without nailgun installed and got...

```
$ ./cbt
(Note: nailgun not found. It makes CBT faster! Try 'brew install nailgun'.)
Starting up nailgun
Recompiling CBT BootstrapScala.java
downloading https://repo1.maven.org/maven2/org/scala-lang/scala-library/2.11.7/scala-library-2.11.7.jar
to /Users/matt/Projects/cbt/bootstrap_scala/cache/2.11.7/scala-library-2.11.7.jar
downloading https://repo1.maven.org/maven2/org/scala-lang/scala-compiler/2.11.7/scala-compiler-2.11.7.jar
to /Users/matt/Projects/cbt/bootstrap_scala/cache/2.11.7/scala-compiler-2.11.7.jar
downloading https://repo1.maven.org/maven2/org/scala-lang/scala-reflect/2.11.7/scala-reflect-2.11.7.jar
to /Users/matt/Projects/cbt/bootstrap_scala/cache/2.11.7/scala-reflect-2.11.7.jar
downloading https://repo1.maven.org/maven2/org/scala-lang/modules/scala-xml_2.11/1.0.5/scala-xml_2.11-1.0.5.jar
to /Users/matt/Projects/cbt/bootstrap_scala/cache/2.11.7/scala-xml_2.11-1.0.5.jar
Recompiling CBT. Detected source changes...
Stopping nailgun
Restarting nailgun
```

I will admit that I killed this before seeing whether or not it continued. Paused for a few minutes so that's probably something to check. But either way, we probably shouldn't be doing nailgun related things if we detect it isn't there.

**Comment by cvogt** [comment]
+1

**Comment by cvogt** [comment]
fixed in 9a46a52f6fb3d7c0d07f3c5aa15fc95289738d52 if I am not mistaken
